OpenSimRT/RealTime/src/experimental/ExternalAccelerationBasedPhaseDetector.cpp

```diff
@@ -0,0 +1,60 @@
+/**
+ * -----------------------------------------------------------------------------
+ * Copyright 2019-2021 OpenSimRT developers.
+ *
+ * This file is part of OpenSimRT.
+ *
+ * OpenSimRT is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * OpenSimRT is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * OpenSimRT. If not, see <https://www.gnu.org/licenses/>.
+ * -----------------------------------------------------------------------------
+ */
+
+#include "ExternalAccelerationBasedPhaseDetector.h"
```

Comment:
Why Extrernal? Maybe use a shorter name.

Reply:
The 'external' was meant as an indication that the acceleration values that dictate the gait phase are measured from an external sensor, instead of being estimations from the model after IK (internal acceleration). Perhaps 'Measured' would be a better name?